### PR TITLE
adding to the cloudformation section

### PR DESCRIPTION
**Pull request by em0ney**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1118,22 +1118,23 @@ CloudFormation
...
 -	🔹Until [2016](https://aws.amazon.com/about-aws/whats-new/2016/09/aws-cloudformation-introduces-yaml-template-support-and-cross-stack-references/), CloudFormation used only an awkward JSON format that makes both reading and debugging difficult. To use it effectively typically involved building additional tooling, including converting it to YAML, but now [this is supported directly](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/template-formats.html).
+- Wherever possible, export relevant Physical IDs from your Stacks by defining [Outputs in your CloudFormation Templates](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/outputs-section-structure.html). They can be returned from `DescribeStack` API calls, and get imported to other Stacks as part of the [recent addition](https://aws.amazon.com/about-aws/whats-new/2016/09/aws-cloudformation-introduces-yaml-template-support-and-cross-stack-references/) of [cross-stack references](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/walkthrough-crossstackref.html)
```
Comment (jlevy) [inline README.md]: Explain "physical ID". Also note AWS does not capitalize the P.

File: README.md [hunk 2]
```diff
@@ -1118,22 +1118,23 @@ CloudFormation
 ### CloudFormation Basics
 
 -	📒 [Homepage](https://aws.amazon.com/cloudformation/) ∙ [Developer guide](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/) ∙ [FAQ](https://aws.amazon.com/cloudformation/faqs/) ∙ [Pricing](https://aws.amazon.com/cloudformation/pricing/) at no additional charge
--	**CloudFormation** offers mechanisms to create and manage entire configurations of many types of AWS resources, using a JSON-based templating language.
+-	**CloudFormation** offers mechanisms to create and update entire 'stacks' comprised of many types of AWS resources.  These CloudFormation stacks are defined in a 'CloudFormation template' which is defined in a [JSON](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-instance.html#aws-properties-ec2-instance-syntax.json) or [YAML](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-instance.html#aws-properties-ec2-instance-syntax.yaml).
```
Comment (jlevy) [inline README.md]: Remove quotes, boldface, and link:

[**stacks**](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/stacks.html)

[**CloudFormation template**](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/template-guide.html)

Grammar: Defined in JSON ... (remove "a")

**Comment by em0ney** [comment]
@jlevy - incorporated feedback and resolved merge conflict

**Comment by jlevy** [comment]
LGTM. Would like one more topic expert or @ThanosBaskous to review as well.

**Comment by ThanosBaskous** [comment]
LGTM. Thank you for your contribution, @em0ney!

**Pull request merged**